ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1835,6 +1855,12 @@ class NoteTextComponent extends React.Component {
 
 		const titleBarDate = <span style={Object.assign({}, theme.textStyle, {color: theme.colorFaded})}>{time.formatMsToLocal(note.user_updated_time)}</span>
 
+		const templatePrompt = <span className={body === '' ? 'fade_in' : 'fade_out'} style={templateStyle}>{_("Nothing here... try inserting a ")}
```

Comment:
Maybe just `const templatePrompt = !this.props.templates.length ? null : ....` as that will remove a few conditionals below.

Reply:
Done.